### ISSUE TITLE
Move To Top shouldn’t reorder the node list.

### ISSUE THREAD

**Issue by KAJed82** [Siccity/xNode#54]
Move to top modifies the node list directly which wreaks havoc when a project is shared between multiple people. Node order should only change when done intentionally. Maintaining a separate list would solve this. Since nodes might also be filtered depending on the implementation it might be useful to have the handler for these behaviours be an interface.

**Comment by Siccity** [comment]
Good point. It'll be a while till I get time to look into it. Feel free to
send a pull request
